binder/appendix.txt

```diff
@@ -1,19 +1,22 @@
 USER root
 ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && \
+    wget -q -O /tmp/lrose.deb https://github.com/NCAR/lrose-core/releases/download/lrose-core-20220222/lrose-core-20220222.ubuntu_18.04.amd64.deb && \
+    apt-get -y install /tmp/lrose.deb && \
+    apt-get clean && \
+    rm -rf /tmp/lrose.deb
+
+USER ${NB_USER}
 RUN CONDA_DIR=/srv/conda && \
     RADARENV=notebook && \
     CONDA_PREFIX=$CONDA_DIR/envs/$RADARENV && \
     export BALTRAD_INSTALL_ROOT=${PWD} && \
     echo "export CONDA_DIR=$CONDA_DIR" >> ~/.profile && \
     echo "export RADARENV=$RADARENV" >> ~/.profile && \
     echo "export CONDA_PREFIX=$CONDA_PREFIX" >> ~/.profile && \
-    ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
-    apt update && \
-    apt-get -y install sudo && \
-    sudo apt-get -y update && \
-    sudo -s && \
-    sudo ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/lrose/build_lrose.sh && \
+    ${CONDA_PREFIX}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/baltrad/install_baltrad.sh && \
     ${CONDA_PREFXI}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/nbstripout_notebooks.sh && \
```

Comment:
```suggestion
    ${CONDA_PREFIX}/bin/bash -l $BALTRAD_INSTALL_ROOT/install/nbstripout_notebooks.sh && \
```

Reply:
One more - sorry!

Reply:
Let me fix this tomorrow, with fresh eyes.